test_sale-keypair.json

```diff
@@ -0,0 +1 @@
+[53,16,113,75,104,41,118,26,85,244,233,233,63,0,128,49,51,119,131,32,161,248,179,154,73,206,247,204,74,72,0,196,12,244,188,146,119,17,207,19,145,222,117,132,184,95,192,254,9,54,73,17,177,97,163,208,187,60,75,195,23,66,56,108]
```

Comment:
add to the contract directory.

Reply:
regenerating a new key and passing to WEN to release on devnet. There is no intent for this program to be deployed. It is a reference implementation.